### PR TITLE
Delete multiple selected workflows

### PR DESCRIPTION
**Pull request by lgalis**
Add a 'Delete' button to the toolbar for the Workflows tab to allow the removal of the selected items in the list.

Screenshot from 2019-05-22 09-40-42

Screenshot from 2019-05-22 09-40-59

Screenshot from 2019-05-22 09-42-48

Screenshot from 2019-05-22 09-43-06

### CODE REVIEW

File: src/smart-components/workflow/workflows.js [hunk 1]
```diff
@@ -45,10 +47,17 @@ const Workflows = ({ fetchRbacGroups, fetchWorkflows, workflows, pagination, his
         title: 'Delete',
         style: { color: 'var(--pf-global--danger-color--100)'	},
         onClick: (event, rowId, workflow) =>
-          history.push(`/workflows/remove/${workflow.id}`)
+          history.push({ pathname: '/workflows/remove-workflow',
+            state: { checkedWorkflows: [ workflow.id ]}
```
Comment (Hyperkid123) [inline src/smart-components/workflow/workflows.js]: I am a bit confused by this change. Is there any reason why we cannot react the id from `match.params`?

File: src/test/smart-components/workflow/workflows.test.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  let mockStore;
+  let initialState;
+
+  beforeEach(() => {
+    initialProps = {};
+    mockStore = configureStore(middlewares);
+    initialState = { workflowReducer: { ...workflowsInitialState, isLoading: false }, groupReducer: { ...groupsInitialState }};
+  });
+
+  afterEach(() => {
```
Comment (Hyperkid123) [inline src/test/smart-components/workflow/workflows.test.js]: Fetch mock is not used anywhere in this test file you can delete `afterEach` method.

Comment (lgalis) [reply src/test/smart-components/workflow/workflows.test.js]: Done

File: src/smart-components/workflow/workflows.js (hunk 1 above)
Comment (lgalis) [reply src/smart-components/workflow/workflows.js]: Moved the cheked ids to props

**Comment by lgalis** [comment]
@Hyperkid123  - feedback addressed

**Pull request merged**